### DISCUSSION TITLE
[Question]: Assetripper extracting more prefabs?

### DISCUSSION THREAD

**Discussion by ningengit** [AssetRipper/AssetRipper#613] [General]
### Describe the issue

So i decompiled a game called "Baldi's basics classic v1.4.3"
in assetripper v0.2.4.2 and a compiled version of assetripper tho i noticed something
in v0.2.4.2 it does not generate additional prefabs while in the compiled one it does more specifically scene gameobjects 
is this a accuracy improvement or something?

**Comment by ds5678** [comment]
Prefab Outlining is a recent improvement, which was enabled by the asset processing framework. It checks the entire project for inlined game objects and outlines them as new prefabs to export.
